media-libs/vidstab/vidstab-1.1.0.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+MY_PN="vid.stab"
+DESCRIPTION="Video stabilization library"
+HOMEPAGE="http://public.hronopik.de/vid.stab/"
+SRC_URI="https://github.com/georgmartius/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+RDEPEND=">=media-video/ffmpeg-3.2.4-r1"
```

Comment:
Does it link to ffmpeg libraries? If so, please add `:=` to link the package to ffmpeg's SONAME and force rebuilding on ffmpeg ABI changes.

Reply:
No it doesn't, ffmpeg uses it, not vice-versa:
$ ldd /usr/lib64/libvidstab.so.1.1 
	linux-vdso.so.1 (0x00007ffcdfd25000)
	libm.so.6 => /lib64/libm.so.6 (0x00007f5f513ca000)
	libgomp.so.1 => /usr/lib/gcc/x86_64-pc-linux-gnu/6.4.0/libgomp.so.1 (0x00007f5f5119c000)
	libc.so.6 => /lib64/libc.so.6 (0x00007f5f50ded000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f5f518f6000)
	libdl.so.2 => /lib64/libdl.so.2 (0x00007f5f50be9000)
	libpthread.so.0 => /lib64/libpthread.so.0 (0x00007f5f509c9000)
$ ldd /usr/bin/ffmpeg|grep vidstab
	libvidstab.so.1.1 => /usr/lib64/libvidstab.so.1.1 (0x00007fe69ebfc000)

Reply:
Well, in that case the dependency should be in ffmpeg, and not here, shouldn't it?

Reply:
Hmm... vidstab ist just a library, it cannot be run in a way, so you are right, should have no RDEPENDs - will fix.